data-manipulation/encryption/hc-128/encrypt-data-using-hc-128-via-wolfssl.yml

```
@@ -0,0 +1,23 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: encrypt data using HC-128 via WolfSSL
+    namespace: data-manipulation/encryption/hc-128
+    author: blaine.stancill@mandiant.com
+    scope: basic block
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027]
+    mbc:
+      - Defense Evasion::Obfuscated Files or Information::Encryption-Standard Algorithm [E1027.m05]
+      - Cryptography::Encrypt Data::HC-128 [C0027.006]
+    references: https://github.com/wolfSSL/wolfssl/blob/master/wolfcrypt/src/hc128.c
+    examples:
+      - 91B08896FBDA9EDB8B6F93A6BC811EC6:0x180003562
+  features:
+    - and:
+      - characteristic: nzxor
+      - number: 0x17 # 23 from line 153: tem0 = rotrFixed((ctx->T[(v)]),23);
+      - number: 0xA # 10 from line 154: tem1 = rotrFixed((ctx->X[(c)]),10);
+      - number: 0x8 # 8 from line 155: tem2 = rotrFixed((ctx->X[(b)]),8);
+      - number: 0x3FC # Compiler optimized size used in ANDs for the 1024 sized buffer
```

```suggestion
      - number: 0x17 = 23 from line 153: tem0 = rotrFixed((ctx->T[(v)]),23);
      - number: 0xA = 10 from line 154: tem1 = rotrFixed((ctx->X[(c)]),10);
      - number: 0x8 = 8 from line 155: tem2 = rotrFixed((ctx->X[(b)]),8);
      - number: 0x3FC = Compiler optimized size used in ANDs for the 1024 sized buffer
```

would you please make this a description?